### DISCUSSION TITLE
Выбор квартиры при доставке "до двери"

### DISCUSSION THREAD

**Discussion by Dragonzcxv** [cdek-it/widget#67] [Q&A] [answered]
Добрый день! 
Судя по примеру отображения вот тут https://github.com/cdek-it/widget/wiki у виджета есть функционал выбора квартиры. Подскажите каким образом вызывать данный функционал? На данный момент пользователь при выборе доставки "до двери" может спокойно выбрать адрес без уточнения квартиры, что не есть хорошо.

**Comment by vermorag** [comment, marked as answer]
Добрый день. Такой функционал появится в 4 версии виджета. Номер запроса на доработку CMS-595. В вики скриншот приведен в качестве примера из рендера 4 версии.
